source/includes/_endpoint-dataset.md

```diff
@@ -565,3 +565,23 @@ PATCH the "expression" attribute to modify. An empty "expression" object, like
 ##### Decks
 
 ##### Permissions
+
+##### Primary key
```

Comment:
Is this a "fragment" rather than a "catalog"? If so, relocate it in the docs. If not, then we should be returning a shoji:catalog rather than entity.